go/pkg/bertyaccount/service.go

```diff
@@ -72,7 +73,10 @@ func (o *Options) applyDefault() {
 	}
 }
 
-func NewService(opts *Options) (Service, error) {
+func NewService(opts *Options) (_ Service, err error) {
+	_, _, endSection := tyber.Section(context.TODO(), opts.Logger, "Initializing AccountService")
+	defer func() { endSection(err, "") }()
```

Comment:
why don't you simply use `defer endSection(err, "")`?

Reply:
see https://play.golang.org/p/ejXulLs39tt